File: concept/04_conditional/ifelse.cpp
#include <iostream>
using namespace std;

int main(){
    int a;
    int b;

    cout << "Enter First value :" << endl;
    cin>>a;

    cout << "Enter Second value :" << endl;
    cin>>b;

    if(a > b){
        cout<< "A is Greater" << endl;
    }else{
        cout<< "B is Greater" << endl;
    }

  int marks = 32;
    // Nested if
  if (marks >= 33) {
    cout<< "You have passed the Exam" << endl;
  } else if (marks < 33) {
    cout<< "You have failed the Exam" << endl;
  } else {
    cout<< "Result not available" << endl;
  }
}

File: concept/04_conditional/if_else.dart
import 'dart:io';

void main() {
  // Taking user Input from Terminal
  int a = stdin.readByteSync();
  int b = stdin.readByteSync();
  if (a > b) {
    print("A is Greater");
  } else {
    print("B is Greater");
  }

  int marks = 33;

  // Nested if
  if (marks >= 33) {
    print("You have passed the Exam");
  } else if (marks < 33) {
    print("You have failed the Exam");
  } else {
    print("Result not available");
  }
}
